docs/docs/resources/azure-ad/AADUser.md

```diff
@@ -11,7 +11,7 @@
 | **Roles** | Write | StringArray[] | The list of Azure Active Directory roles assigned to the user. | |
 | **UsageLocation** | Write | String | The country code the user will be assigned to | |
 | **LicenseAssignment** | Write | StringArray[] | The account SKU Id for the license to be assigned to the user | |
-| **Password** | Write | PSCredential | The password for the account. The parameter is a PSCredential object, but only the Password component will be used | |
+| **Password** | Write | PSCredential | The password for the account. The parameter is a PSCredential object, but only the Password component will be used. If Password is not supplied for a new resource a new random password will be generated. Property will only be used when creating the user and not on subsequent updates. | |
```

Comment:
This file is generated based on the information in the schema. Could you please also update the description of the Password parameter in the schema.mof, so this change isn't rolled back when the file is generated again.